### ISSUE TITLE
Flat hierarchy goes to root without changing breadcrumb

### ISSUE THREAD

**Issue by tjomson** [git-truck/git-truck#721]
It should probably just stay in the current directory

![image](https://github.com/git-truck/git-truck/assets/23435481/f2260d43-0fdc-49c8-9a81-32fab246ad1f)

**Comment by joglr** [comment]
So it should flatten the current folder instead of the root?

**Comment by tjomson** [comment]
Yeah. I think the problem is that the flattening happens on the commit (Chart.tsx:75), such that when the hierarchy is partitioned, there is no folder structure, so all files are logically placed in the root (Chart.tsx:87)
